eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/SlugConverter.php

```diff
@@ -74,7 +73,6 @@ class SlugConverter
                     'endline_search_normalize',
                     'tab_search_normalize',
                     'specialwords_search_normalize',
-                    'punctuation_normalize',
```

Comment:
@glye: In your description you say that the 'urlalias_compat' need no change. But this line is actually changing it. And was not part of the original pull request.